src/UKHO.ERPFacade.CleanUp.WebJob/Services/CleanUpService.cs

```diff
@@ -25,25 +26,30 @@ public CleanUpService(ILogger<CleanUpService> logger,
         }
 
         public void CleanUpAzureTableAndBlobs()
-        {            
-            CleanUpBlobsAndTablesRelatedToEESEvent();
+        {
+            CleanUpEvents(Constants.S57EventTableName, Constants.S57EventContainerName);
         }
 
-        //Private methods       
-        private void CleanUpBlobsAndTablesRelatedToEESEvent()
+        private void CleanUpEvents(string tableName, string eventContainerName)
         {
-            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all EES entities from azure table");
-            var entities = _azureTableReaderWriter.GetAllEntityForEESTable();
+            _logger.LogInformation(EventIds.FetchEESEntities.ToEventId(), "Fetching all records from azure table {0}", tableName);
+
+            var entities = _azureTableReaderWriter.GetAllEntities(tableName);
+
             foreach (var entity in entities)
             {
-                if (!entity.RequestDateTime.HasValue)
+                if (entity["RequestDateTime"] == null)
                     continue;
-                TimeSpan timediff = DateTime.Now - entity.RequestDateTime.Value;
+
+                TimeSpan timediff = DateTime.Now - Convert.ToDateTime(entity["RequestDateTime"].ToString());
+
                 if (timediff.Days > int.Parse(_erpFacadeWebjobConfig.Value.CleanUpDurationInDays))
                 {
-                    Task.FromResult(_azureTableReaderWriter.DeleteEESEntity(entity.CorrelationId));
-                    _logger.LogInformation(EventIds.DeletedContainerSuccessful.ToEventId(), "Deleting container : {0}", entity.CorrelationId);
-                    _azureBlobEventWriter.DeleteContainer(entity.CorrelationId.ToLower());
+                    Task.FromResult(_azureTableReaderWriter.DeleteEntity(entity["CorrelationId"].ToString(), tableName));
+
+                    _logger.LogInformation(EventIds.DeletedContainerSuccessful.ToEventId(), "Deleting directory {0} from {1} container", entity["CorrelationId"].ToString(), eventContainerName);
```

Comment:
Can we use variable names in string interpolation to keep it consistent with other logs.